src/sce-elf-defs.h

```diff
@@ -91,6 +91,21 @@ typedef struct SCE_TYPE(sce_module_imports_short) {
 	SCE_PTR(const void **) var_entry_table;		/* Pointer to array of data pointers to write to */
 } SCE_TYPE(sce_module_imports_short);
 
+typedef struct SCE_TYPE(sce_process_param_old) {
+	uint32_t size;                          /* 0x30 */
+	uint32_t magic;                         /* PSP2 */
+	uint32_t version;                       /* Unknown, but it could be 6 */
+	uint32_t fw_version;                    /* SDK vsersion */
+	SCE_PTR(const char *) main_thread_name; /* Thread name pointer*/
+	int32_t main_thread_priority;           /* Thread initPriority */
+	uint32_t main_thread_stacksize;         /* Thread stacksize*/
+	uint32_t main_thread_attribute;         /* Unknown */
+	SCE_PTR(const char *) process_name;     /* Process name pointer */
+	uint32_t process_preload_disabled;      /* Module load inhibit */
+	uint32_t main_thread_cpu_affinity_mask; /* Unknown */
+	SCE_PTR(const void *) sce_libc_param;   /* SceLibc param pointer */
+} SCE_TYPE(sce_process_param_old);
+
 typedef struct SCE_TYPE(sce_process_param) {
```

Comment:
this param may need a document about that this param only can use process_param_version >= 0x1692000

Reply:
done